app/views/manifests/show.html.erb

```diff
@@ -57,13 +56,13 @@
       <dt>U.S. EPA Federal Waste Codes</dt>
       <dd>
         <% m["epa_waste_codes"].each do |wc| %>
-          wc
+          wc<% if wc != m["epa_waste_codes"].last %>, <% end %>
```

Comment:
I'd prefer to have this logic in a helper method rather than having it in the view. Would you be cool w adding that or would you prefer than one of us does it?

http://mixandgo.com/blog/the-beginner-s-guide-to-rails-helpers